bot/cogs/study.py

```diff
@@ -187,16 +194,15 @@ async def _definition(self, ctx: Context, word: str) -> dict:
         out = {}
         if not lists:
             return
-        for a in types:
-            reg = str(lists[types.index(a)])
+        for a_type in types:
+            reg = str(lists[types.index(a_type)])
```

Comment:
It would be best to do `for type in types` but since `type` is a python reserved object it wouldn't be wise to overwrite it, so instead we could use `_type`
```suggestion
        for `_type` in types:
            reg = str(lists[types.index(_type)])
```

Reply:
Done